addons/sale_stock/sale_stock.py

```diff
@@ -271,6 +271,17 @@ def _run_move_create(self):
             vals.update({'sequence': self.sale_line_id.sequence})
         return vals
 
+    @api.model
+    def create(self, vals):
+        procurement = super(ProcurementOrder, self).create(vals)
+        move = procurement.move_dest_id
+        if move.group_id:
+            order = self.env['sale.order'].search([('procurement_group_id', '=', move.group_id.id)])
```

Comment:
I don't like the fact that the child record is looking for his father, in order to send himself a message. I'll rework this part